fix(l10n): Less jargon in source strings

kra-mo opened the pull request:
Resolves #53775

## Summary
Removes:
- “enable” for settings
- “settings” for settings sections
- “all” when there is no obvious subset of items
- “show” and “open” for navigation actions
- “changes” for applying/discarding
- “to clipboard” when copying
- Explaining things that cannot happen
- Explaining things twice, right below each other
- Unnecessary technical jargon
- Text that sounds like marketing copy and serves no other purpose

If there is anything in particular that was there for a specific reason, please do tell me.

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [x] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [x] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required

skjnldsv commented on apps/federatedfilesharing/src/components/PersonalSettings.vue:
```diff
@@ -146,7 +146,7 @@ export default {
 		},
 		copyLinkTooltip() {
-			return this.isCopied ? t('federatedfilesharing', 'Cloud ID copied to the clipboard') : t('federatedfilesharing', 'Copy to clipboard')
+			return this.isCopied ? t('federatedfilesharing', 'Cloud ID copied to the clipboard') : t('federatedfilesharing', 'Copy')
```
Wanna remove that section too?

```suggestion
			return this.isCopied ? t('federatedfilesharing', 'Cloud ID copied') : t('federatedfilesharing', 'Copy')
```

kra-mo replied:
Aha! Only searched for "to clipboard", not "to the clipboard".

Hold on… :)

kra-mo replied:
Surprisingly, it seems like that was the only place it appeared haha

skjnldsv reviewed (approved):
Looking good, let's see tests results!

susnux reviewed (commented):
Would be nice to split such changes, at least into commits but better into separate PRs.
Because you not only change l10n strings as the commit and the PR suggests, but also remove user notifications from the actions.

So e.g. on PR with only the string change + one PR with removing notifications.

susnux commented on apps/files/src/filters/TypeFilter.ts:
```diff
@@ -73,7 +73,7 @@ const getTypePresets = async () => [
 	{
 		id: 'image',
 		// TRANSLATORS: This is for filtering files, e.g. PNG or JPEG, so photos, drawings, or images in general
-		label: t('files', 'Photos and images'),
+		label: t('files', 'Images'),
```
Not sure about this one, we had a discussion with translators in the past about this being not clear.

jancborchardt replied:
@rakekniven do you remember what this discussion was about? Other apps like e.g. Google Files on Android, Files on iOS use just "Images".

susnux replied:
Maybe it was translated to "Abbilder" in German? Something like that?

In that case the translator comment should be enough to have context for proper translations, no?

susnux commented on apps/files/src/views/Settings.vue:
```diff
@@ -113,22 +113,20 @@
 				:checked="userConfig.show_dialog_file_extension"
 				@update:checked="setConfig('show_dialog_file_extension', $event)">
-				{{ t('files', 'Show a warning dialog when changing a file extension.') }}
+				{{ t('files', 'Warn me before changing a file extension') }}
```
I do not think we use this tone somewhere else, IIRC all apps and other places do not speak to you in first person (omitting the `me`). Which I imagine could be a translation problem for consistency (because translations use the formal language, in the default German this is hard to translate (event though we have two different translations for German, a formal ("Sie") and a informal one - but the formal one if the Nextcloud official)

susnux replied:
(just saying that this maybe needs coordination with out translators as a change of language)

susnux commented on apps/files/src/views/Settings.vue:
```diff
@@ -137,39 +135,39 @@
 					</dt>
 					<dd class="shortcut-description">
-						{{ t('files', 'Favorite or remove a file from favorites') }}
+						{{ t('files', 'Add or remove favorite') }}
```
Is it possible in English to use the zeroarticle here? I mean instead of

```suggestion
						{{ t('files', 'Add or remove a favorite') }}
```

susnux replied:
yeah than I am fine with it - my German brain does not like the *zero article* ^^

susnux commented on apps/files_sharing/src/views/SharingTab.vue:
```diff
@@ -230,9 +230,9 @@ export default {
-			additionalSharesHelpText: t('files_sharing', 'Shares that are not part of the internal or external shares. This can be shares from apps or other sources.'),
+			internalSharesHelpText: t('files_sharing', 'Share files within your organization. Recipients who can already view the file can also use this link for easy access.'),
+			externalSharesHelpText: t('files_sharing', 'Share files with others outside your organization via public links and email addresses. You can also share to Nextcloud accounts on other instances using their federated cloud ID.'),
+			additionalSharesHelpText: t('files_sharing', 'Shares from apps or other sources which are not internal or external.'),
```
I think the last part is a bit confusing, if its only me then you should add a comment for translators as there would be likely also confused ones :sweat_smile:
At least for me its easier readable if

```suggestion
			additionalSharesHelpText: t('files_sharing', 'Shares from apps or other sources which are not included in internal or external shares.'),
```

To make clear that the last part is about the two other sections in the UI and not about the `sources`.

susnux reviewed (commented):
I am not sure about the removed cancellation notifications, as I remember we had requests to explicitly add those to make clear whats going on.
Otherwise just a few comments.

jancborchardt reviewed (commented):
Good on removing the overbearing notifications. For the record @susnux yes we did talk about it and also had a request for it, but in practice this proves to be super annoying and results in notification fatigue and missing actually important notifications. :\

jancborchardt commented on apps/files/src/components/FilesNavigationSearch.vue:
```diff
@@ -72,13 +72,13 @@ const searchLabel = computed(() => {
 						<NcIconSvgWrapper :path="mdiMagnify" />
 					</template>
-					{{ t('files', 'Filter and search from this location') }}
+					{{ t('files', 'Search this location') }}
```
Similarly here:
```suggestion
					{{ t('files', 'Search here') }}
```

jancborchardt reviewed (commented):
Really good also on the general simpler wording @kra-mo :) makes everything much easier to read.

kra-mo commented on apps/files/src/views/Settings.vue:
```diff
@@ -8,7 +8,7 @@
 		@update:open="onClose">
 		<!-- Settings API-->
-		<NcAppSettingsSection id="settings" :name="t('files', 'Files settings')">
+		<NcAppSettingsSection id="settings" :name="t('files', 'General')">
```
This is not the main title, it is the first section. You just experienced how confusing it was :)

kra-mo replied:
No, not the actual title, it is this:

<img width="540" height="351" alt="Image" src="https://github.com/user-attachments/assets/27f21dda-9042-40e8-937f-74bc585e4db6" />

kra-mo replied:
Yes, currently it should be the same string, but after the changes, it will be two separate ones.

kra-mo replied:
I don't know of any other apps that have "{app name} settings" as a settings *section* and Contacts for example already has a "General" one (although they also have "settings" in the name).

kra-mo replied:
Mail also has "General".

joshtrichards replied:
So for *Files* we currently have [Button: *Files settings*] --> [Heading: *Files settings*] -> [1st Section: *Files settings*].

As a comparison to another shipped app...

For *Photos* we currently have [Button: *Photos settings*] --> [Heading: *Photos settings*] -> [1st Section: *View*].

With this PR for *Files* we'd have [Button: *Files settings*] --> [Heading: *Files settings*] -> [1st Section: *General*].

That works for me.

kra-mo commented on apps/systemtags/src/components/SystemTagPicker.vue:
```diff
@@ -89,7 +89,7 @@
 			<!-- Note -->
 			<div class="systemtags-picker__note">
 				<NcNoteCard v-if="!hasChanges" type="info">
-					{{ canEditOrCreateTag ? t('systemtags', 'Select or create tags to apply to all selected files'): t('systemtags', 'Select tags to apply to all selected files') }}
```
On purpose, it is repeated information as it is already explained somewhere else in the same dialog.

kra-mo commented on apps/files/src/utils/davUtils.ts:
```diff
@@ -29,7 +29,7 @@ export function humanizeWebDAVError(error: unknown) {
 				return t('files', 'This operation is forbidden')
 			} else if (status === 500) {
-				return t('files', 'This directory is unavailable, please check the logs or contact the administrator')
+				return t('files', 'This folder is unavailable, please try again later or contact the administrator')
```
Fair enough but not part of this PR. I'd make it a separate one if you want to change it, it'd be a bigger initiative to merge the two ways of phrasing since it is used *a lot*, across apps.

kra-mo replied:
> Better to only change strings once to not double the work for them.

That is fair, yeah. I can change this one string as they are already inconsistent. But I do not want this PR to get out of hand and become an unrebasable mess :D

kra-mo committed:
fix(l10n): Less jargon in source strings

This removes:
- “enable” for settings
- “settings” for settings sections
- “all” when there is no obvious subset of items
- “show” and “open” for navigation actions
- “changes” for applying/discarding
- “to clipboard” when copying
- Explaining things that cannot happen
- Explaining things twice, right below each other
- Unnecessary technical jargon
- Text that sounds like marketing copy and serves no other purpose

Signed-off-by: kramo <git@kramo.page>

susnux commented on apps/files/src/components/FilesListTableHeaderActions.vue:
```diff
@@ -305,16 +305,16 @@ export default defineComponent({
 
 				// Show success message and clear selection
-				showSuccess(this.t('files', '"{displayName}" batch action executed successfully', { displayName }))
+				showSuccess(this.t('files', '{displayName}: done', { displayName }))
```
It is just a concatenation of the displayname, it could also be "Restore: done".
We do not have knowledge of the display name here.

But this should also only be shown if the action does not provide a message themself, so  I would say this is fine here and use proper messages in the individual actions.

kra-mo commented:
> @kra-mo Please refrain from marking the conversation as resolved if I still have unanswered questions for you. Once the questions have been clarified, it is acceptable; otherwise, it is unprofessional.

Sorry, I expected my answers to answer your questions.

kra-mo commented:
> Related: #53775 for:
> 
> https://github.com/nextcloud/server/blob/e027cd44e4ea1ef853bedde64627e46c336a0c9c/apps/files_sharing/src/views/SharingTab.vue#L273-L274

Yeah, fair. I updated those strings.

susnux commented:
/compile rebase

kra-mo commented:
> Testes in `cypress/e2e/files/search.cy.ts` need to be adjusted

Could you help me with that? I tried to look but couldn't find anything.

And the e2ee one seemed to be failing for other PRs too, but maybe that was unrelated?

rakekniven committed:
fix: Update tests

Signed-off-by: rakekniven <2069590+rakekniven@users.noreply.github.com>

nextcloud-command committed:
chore(assets): Recompile assets

Signed-off-by: nextcloud-command <nextcloud-command@users.noreply.github.com>
